cherryontech/JourneyWise
# Mobile Responsiveness (part 1)

## Pull Request

### Description
I was finally able to make some headway with the mobile responsiveness layout for the home page. 

### Related Issue
[JW-75](https://marisolsaumi.atlassian.net/browse/JW-75?atlOrigin=eyJpIjoiOWI3ZDdiNzUzMmI4NDAwMWJjYTk4MWNmMzQ0MDBiMjkiLCJwIjoiaiJ9) : Responsiveness

### Description of Changes
* Worked on making the homepage look like the Figma mobile layout. 
* Homepage should look better on mobile. 
* There are still issues. We need to figure out the language bar/nav bar, as well as why the collage photo does not show up (the collage is there...you can see where it is supposed to be...the actual photo is just ghosting us right now). 
* Also need to possibly get new yellow number dots from Rocio for the homepage guide.  

### Checklist
- [ ] I have updated the documentation or comments where necessary.

### Testing Instructions
* Please view these changes on a mobile device. 
* If you are viewing this website on a desktop, use Google Chrome's dev tools to preview these changes. You can open our deployment preview, right click on the page and click "inspect", and at the top of the dev tools panel click the two rectangles (they look like a laptop and a phone kind of, its to the left of "Elements" and ensure that the browser size is 480px and below. 

### Screenshots (if applicable)
<img width="1414" alt="Screenshot 2024-06-02 at 3 13 52 PM" src="https://github.com/cherryontech/JourneyWise/assets/122942301/2d55e05f-3aed-4cb7-8709-08f0b5f1dea8">
<img width="347" alt="Screenshot 2024-06-02 at 3 16 00 PM" src="https://github.com/cherryontech/JourneyWise/assets/122942301/9b647950-dd4f-41a1-af60-d03304fdfeda">
<img width="343" alt="Screenshot 2024-06-02 at 3 16 07 PM" src="https://github.com/cherryontech/JourneyWise/assets/122942301/f7c01eed-2f5c-4a51-9777-db9d250b3326">
<img width="340" alt="Screenshot 2024-06-02 at 3 16 16 PM" src="https://github.com/cherryontech/JourneyWise/assets/122942301/570e4c27-8627-4e16-9609-87394370fe37">
<img width="338" alt="Screenshot 2024-06-02 at 3 16 24 PM" src="https://github.com/cherryontech/JourneyWise/assets/122942301/8dd1f710-82f7-4704-b517-1c8168ec8c5d">
<img width="354" alt="Screenshot 2024-06-02 at 3 16 32 PM" src="https://github.com/cherryontech/JourneyWise/assets/122942301/0d9892d6-ff8d-4755-983b-c7fadc607687">
<img width="352" alt="Screenshot 2024-06-02 at 3 16 41 PM" src="https://github.com/cherryontech/JourneyWise/assets/122942301/233c23e2-8a23-48d3-a02f-7ac4399a9699">


### Gotchas/Learnings
* Too many to count with Tailwind! I had to add many custom css files to really get this responsiveness to work. 


### Additional Notes (Optional)
* Please note that this still isn't done. I wanted to merge in the changes but not overwhelm our existing code.
